### PR TITLE
Remove pipeline consumer

### LINKED ISSUE #65: Remove pipeline consumer
Removing the pipeline consumer part of fiaas-deploy-daemon has been planned for a while. Tracking it here for visibility.

The first step of this is implemented in #63

### PR DESCRIPTION
**Pull request by oyvindio**
The pipeline consumer is no longer in use. I think this should remove all traces of it, except the `--disable-pipeline-consumer` config flag, which now does nothing. I'll remove that in a followup PR to make this easier to roll out.

Fixes #65

### CODE REVIEW

**Comment by oyvindio** [comment]
I created #100 to fix the CI build for master. I'll merge it here too to get the e2e tests going and rebase later

File: docs/operator_guide.md [hunk 1]
```diff
@@ -123,10 +123,6 @@ In order for this to work DNS entries for the resulting hosts needs to be create
 
 If you wish to expose certain environment variables to every application in your cluster, define them here. Regardless of how a variable is passed in (option, environment variable or in a config file), it must be specified as `<key>=<value>`.
 
-### blacklist / whitelist
-
-This is currently only used for FINN, and will go away soon.
```
Comment (mortenlj) [inline docs/operator_guide.md:128]: There are a couple other options (`proxy` and `infrastructure` if memory serves) mentioned in this file that is either no longer used, or needs updated descriptions.

File: fiaas_deploy_daemon/config.py [hunk 2]
```diff
@@ -276,42 +261,6 @@ def _resolve_env(self):
...
-                "{} is not set to a port-number, but instead {!r}. Unable to resolve service {}".format(port_key, port,
-                                                                                                        service_name))
-        return host, port
-
-    @staticmethod
-    def _resolve_service_from_srv_record(service_name, port_name):
-        service = service_name.replace("_", "-")
-        port = (port_name if port_name else service_name).replace("_", "-")
-        srv = "_{}._tcp.{}".format(port, service)
-        answers = dns.resolver.query(srv, 'SRV')
```
[the remark this answers is not in the document]
Comment (oyvindio) [reply fiaas_deploy_daemon/config.py:311]: I think dnspython is still used here, but this is just a script: https://github.com/fiaas/fiaas-deploy-daemon/blob/remove-pipeline-consumer/bin/ingress_check.py#L24

File: docs/operator_guide.md (hunk 1 above)
Comment (oyvindio) [reply docs/operator_guide.md:128]: - `proxy` will be used for usage tracking if set and usage tracking is enabled
- `infrastructure` is still used to set `FIAAS_INFRASTRUCTURE` in deployed application's environments. I'm not sure if any applications rely on this or not, so I would rather not remove it as part of this PR

Comment (oyvindio) [reply docs/operator_guide.md:128]: I've updated the docs for the config parameter to reflect this

File: fiaas_deploy_daemon/config.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mortenlj) [reply fiaas_deploy_daemon/config.py:311]: Might just be incentive to delete the script. Is it used anymore? I wrote it to debug/monitor issues around certificate replacement after we made some changes to the TLS implementation. But that can be dealt with in a separate PR.

File: docs/operator_guide.md (hunk 1 above)
Comment (mortenlj) [reply docs/operator_guide.md:128]: 👍

**Pull request merged**